.github/workflows/build.yml

```diff
@@ -340,6 +341,15 @@ jobs:
         run: |
           ve1/bin/pr-comment ${{ steps.check_pr_content.outcome }} ${{ steps.run-verifier.outcome }} ${{ steps.check_report.conclusion }}
 
+      # Note(komish): This step is a temporary fix for the metrics step in the next job
+      # which expects the PR comment to exist at the specified filesystem location.
+      - name: Encode PR Comment for Metrics
+        id: encode_pr_comment
+        if: ${{ always() && needs.setup.outputs.run_build == 'true' }}
+        run: |
+          commentBase64=$(base64 --wrap=0 ./pr/comment)
```

Comment:
Use `${{ steps.pr_comment.outputs.message-file }}` instead of hard-coded `./pr/comment`

Reply:
Done!